src/components/Article.js

```diff
@@ -1,22 +1,23 @@
 import { h, Fragment } from 'preact'
 import { memo } from 'preact/compat'
-import { useState, useRef, useEffect, useLayoutEffect } from 'preact/hooks'
+import { useState, useRef, useEffect, useLayoutEffect, useContext } from 'preact/hooks'
 import {
   ReferencePreview, ArticleToc, ArticleLanguage,
   ArticleMenu, ArticleFooter, Loading, QuickFacts,
   Error, Gallery, Table
 } from 'components'
 import {
   useArticle, useI18n, useSoftkey,
-  useArticlePagination, useArticleLinksNavigation, useArticleTextSize,
+  useArticlePagination, useArticleLinksNavigation,
   usePopup, useTracking
 } from 'hooks'
 import { articleHistory, confirmDialog, goto, viewport } from 'utils'
+import { FontContext } from 'contexts'
 
 const ArticleBody = memo(({ content }) => {
   return (
     <div
-      class='article-content adjustable-font-size'
+      class={'article-content'}
```

Comment:
You don't need an expression between `{}` here because the class is a string literal. 

This applies in many places in this PR.

Reply:
Good catch, this is left over from when we were placing `fontSizeClass` in each element directly